server/utils/log-internal-error.js

```diff
@@ -0,0 +1,7 @@
+const logInternalError = (error) => {
+  const errorText = `Operation failed!\n${
+    error.config.method} to ${error.config.url}\ncode: ${
+    error.code}`;
+  console.log(errorText);
+};
+export default logInternalError;
```

Comment:
Eh, endline. :sweat_smile:

Reply:
Hmm, shouldn't `npm run lint` check for it?

Reply:
Hmm, shouldn't `npm run lint` check for it?

Reply:
I am not sure, this is a new repository, I've not even installed it yet! :sweat_smile: 
If not npm something needs to detect and fail travis for it, most likely it would be `openwisp-utils chec[qa]` script that needs to be added here!

Reply:
I've added `eof-last` rule to `eslintrc.json`